src/api/Game.js

```diff
@@ -0,0 +1,38 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const getGameId = async (gameName) => {
+  try {
+    const headers = {
+      method: 'POST',
+      body: JSON.stringify({ name: gameName }),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    };
+    const response = await fetch(`${url}/games`, headers);
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    return null;
+  }
+};
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216096471-832a37f3-3644-45d4-a645-ff7a0409ddee.png)

**[Optional]**

- [ ] You don't have commands in these sections which are shown in the image above :arrow_up: Kindly put the following commands in the following sections :+1:

`npm install` in Install Section
`npm start` in the Usage Section
`npm test` in the Test Section